src/query/api/v1/handler/database/create_test.go

```diff
@@ -184,7 +184,8 @@ func testLocalType(t *testing.T, providedType string, placementExists bool) {
 						"runtimeOptions": null,
 						"schemaOptions": null,
 						"coldWritesEnabled": false,
-						"extendedOptions": null
+						"extendedOptions": null,
```

Comment:
nit; add one for non nil opts?

Reply:
Not setting this on the database/create endpoint yet so leaving as-is. Once we hook this up for real (i.e. when the namespace/mark_ready endpoint is available), then we can update these expectations to check for `Initializing`. I'll update the namespace/add_test.go though since you can set the field via that endpoint.